cmd/flux/create_source_oci.go

```diff
@@ -156,6 +160,20 @@ func createSourceOCIRepositoryCmdRun(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if secretName := sourceOCIRepositoryArgs.verifySecretRef; secretName != "" {
```

Comment:
If only the verify provider is specified then it's keyless, so we need to handle that as well. Docs here: https://fluxcd.io/flux/components/source/ocirepositories/#keyless-verification

Reply:
fixed, thanks!!!